src/plugins/share/public/plugin.test.ts

```diff
@@ -12,6 +12,14 @@ import { CoreStart } from '@kbn/core/public';
 import { coreMock } from '@kbn/core/public/mocks';
 import { anonymousAccessMock } from '../common/anonymous_access/index.mock';
 
+const getConfig = () => {
+  return {
+    new_version: {
+      enabled: false,
+    },
+  };
+};
```

Comment:
We should be able to pass the config as an object directly, since we aren't validating that this function got invoked